doc/tutorial.rst

```diff
@@ -1031,46 +1032,9 @@ container. The simplest is to manually invoke :code:`ch-run` in the
    existing user namespace :code:`’2256`; rather, it has re-used the namespace
    ID :code:`’2256`.
 
-Another is to use the :code:`ch-ssh` wrapper program, which adds
-:code:`ch-run` to the :code:`ssh` command implicitly. It takes the
-:code:`ch-run` arguments from the environment variable :code:`CH_RUN_ARGS`,
-making it mostly a drop-in replacement for :code:`ssh`. For example::
-
-  $ export CH_RUN_ARGS="/var/tmp/hello.sqfs --"
-  $ ch-ssh localhost stat -L --format='%i' /proc/self/ns/user
-  4026532256
-  $ ch-ssh -t localhost /bin/bash
-  > stat -L --format='%i' /proc/self/ns/user
-  4026532256
-
-:code:`ch-ssh` is available inside containers as well, in :code:`/usr/bin` via
-bind-mount, if the image has a dummy file at :code:`/usr/bin/ch-ssh`::
-
-  $ export CH_RUN_ARGS="/var/tmp/hello.sqfs --"
-  $ ch-run /var/tmp/hello.sqfs -- /bin/bash
-  > stat -L --format='%i' /proc/self/ns/user
-  4026532256
-  > ch-ssh localhost stat -L --format='%i' /proc/self/ns/user
-  4026532258
-
-This also demonstrates that :code:`ch-run` does not alter most environment
-variables.
-
-.. warning::
-
-   1. :code:`CH_RUN_ARGS` is interpreted very simply; the sole delimiter is
-      spaces. It is not shell syntax. In particular, quotes and backslashes
-      are not interpreted.
-
-   2. Argument :code:`-t` is required for SSH to allocate a pseudo-TTY and
-      thus convince your shell to be interactive. In the case of Bash,
-      otherwise you’ll get a shell that accepts commands but doesn’t print
-      prompts, among other other issues. (`Issue #2
-      <https://github.com/hpc/charliecloud/issues/2>`_.)
-
-A third approach may be to edit one’s shell initialization scripts to check
-the command line and :code:`exec(1)` :code:`ch-run` if appropriate. This is
-brittle but avoids wrapping :code:`ssh` or altering its command line.
+Another may be to edit one's shell initialization scripts to check the command
+line and :code:`exec(1)` :code:`ch-run` if appropriate. This is brittle but
+avoids wrapping :code:`ssh` or altering its command line.
```

Comment:
Would a comment, here or in FAQ, that points to a previous tag/commit ch-ssh file  as an example be helpful?

Something like: "Another may be to implement a ssh wrapper, see since removed (link) as an example."?